modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -47,34 +41,38 @@ anypoint.platform.gatekeeper=flexible
 
 === Security Level Definitions
 
+You can apply one of the listed security levels to Gatekeepr:
```

Comment:
Typo (Gatekeepr) and also a colon should not precede a subhead.

FIRST SUGGESTION...but I now think that this should be a table, or a list that pairs the equivalent levels: `false` and `disabled`, `true` and `strict`, and `flexible`:
_I suggest you intro the security levels in this opening sentence first, and then precede to the subtopics for each.
Alternatively, you could make this a definition list or a bullet list instead of using subtopics._

Reply:
See below

Reply:
Done! Thanks!